kolibri/plugins/learn/assets/src/my_downloads/views/DownloadsList/index.vue

```diff
@@ -290,32 +388,44 @@
         return this.selectedDownloads.indexOf(id) !== -1;
       },
       removeResource(download) {
```

Comment:
We are triggering this method when pressing the 'remove' button on a single download, but this doesn't actually seem to be doing anything, except adding it to the selection list - so this seems like the remove button is only doing the same as selecting the item now?

Seems like this should actually be:
```
this.$emit('removeResources', [download]);
```

If this was needed to get rid of the confirmation dialog, we might need two events, one for `removeResourcesWithConfirmation` which the `removeResources` method uses, and one for `removeResourcesWithoutConfirmation` that is used here.

Reply:
Updated to rename functions for more clarity around their purposes